lib/mp4/caption-parser.js

```diff
@@ -452,4 +460,6 @@ var CaptionParser = function() {
   this.reset();
 };
 
+CaptionParser.prototype = new Stream();
```

Comment:
Had to make this a stream as we don't have a better way to get listeners right now.